test/TEST-09-ISSUE-2691/test.sh

```diff
@@ -6,7 +6,6 @@ TEST_DESCRIPTION="https://github.com/systemd/systemd/issues/2691"
 TEST_NO_NSPAWN=1
 
 . $TEST_BASE_DIR/test-functions
-SKIP_INITRD=yes
```

Comment:
It seems that `QEMU_TIMEOUT` should be tweaked accordingly. Otherwise the tests can time out as in https://objectstorage.prodstack4-5.canonical.com/v1/AUTH_77e2ada1e7a84929a74ba3b87153c0ac/autopkgtest-bionic-pitti-systemd-semaphore/bionic/amd64/s/systemd-upstream/20180719_134229_b652b@/log.gz from https://github.com/systemd/systemd/pull/9655.

Reply:
Agreed, I was debating with myself whether we should simply set the global QEMU_TIMEOUT in test-functions from infinity to say 10min or if each individual test should have to set it explicitly. Thoughts?

Reply:
It's been hard to predict how long it takes for each test to finish since 3e7aa2edcdc893e1 was merged, so I wouldn't set the global timeout explicitly. In the tests where `QEMU_TIMEOUT` is used, I think it should be enough to increase it by, say, 30-60s.

Reply:
Are the tests run regularly on Debian? Does anybody look at the results? If not, I think it'd be better to revert this commit. At least until we figure out how to deal with the timeouts. @martinpitt what do you think? The failures are a bit annoying and I'm not sure we want to reach the point where the results are simply ignored because the tests are flaky.

Reply:
Why do you want to revert the commit? As I wrote in the commit message, I need an initrd so the tests can successfully run in a qemu based autopkgtest  (The debian kernel has ext4 support built as a module and would fail with a kernel oops otherwise).
The issue about QEMU_TIMEOUT is orthogonal to this one.

Reply:
Because the tests now fails regularly on Ubuntu CI due to the timeout issue I mentioned earlier.

Reply:
Which test(s) specifically? The usage of an initrd should not considerably slow down the setup stage...

Reply:
```
test/TEST-08-ISSUE-2730/test.sh:QEMU_TIMEOUT=180
test/TEST-09-ISSUE-2691/test.sh:QEMU_TIMEOUT=90
test/TEST-17-UDEV-WANTS/test.sh:QEMU_TIMEOUT=180
test/TEST-18-FAILUREACTION/test.sh:QEMU_TIMEOUT=180
test/TEST-19-DELEGATE/test.sh:QEMU_TIMEOUT=180
```
Is it TEST-09-ISSUE-2691 or are other test (regularly) failing as well? Why not just bump that to 180s?